### ISSUE TITLE
Add Purchase Intent segments for August campaigns

### ISSUE THREAD

**Issue by moritzhaller** [brave/brave-browser#10891]
Add Purchase Intent segments for August campaign (one-off change and not required for 1.11.x nor 1.13.x)

**Comment by jsecretan** [comment]
I added the campaign with 3 creative sets for each of the 3 segments for this test.  This is only necessary for the US.

1) Go to https://www.pcgamer.com.  Make sure you receive a creative mentioning either "Alpha Influencers", "PC Purists" or "Mainstream Enthusiasts".

2)  Perform a  google search for "AMD".  Make sure you receive a creative mentioning "Alpha Influencers".

3) Perform a  google search for "Gaming Computers".  Make sure you receive a creative mentioning "PC Purists". 

4)  Perform a  google search for "Mobile Games".  Make sure you receive a creative mentioning "Mainstream Enthusiasts".

**Comment by kjozwiak** [comment]
Closing as https://github.com/brave/brave-core/pull/6202 was merged into `1.12.x` 👍

**Comment by LaurenWags** [comment]
Verified passed with

```
Brave | 1.12.108 Chromium: 84.0.4147.105 (Official Build) (64-bit)
-- | --
Revision | a6b12dfad6663f13a7e16e9a42a6a4975374096b-refs/branch-heads/4147@{#943}
OS | macOS Version 10.14.6 (Build 18G3020)
```

* Verified test plan from https://github.com/brave/brave-browser/issues/10891#issuecomment-665766574  (used some different keywords to get additional coverage)

<details>
<summary> Case 1 - visiting https://www.pcgamer.com/</summary>

```
[3573:775:0807/090326.862098:VERBOSE1:ads_impl.cc(732)] Extracting purchase intent signal from visited URL
```

**`Entry in client.json`**:

```
  "purchaseIntentSignalHistory": {
    "Alpha Influencers": [
      {
        "timestamp_in_seconds": 1596805406,
        "weight": 1
      }
    ],
    "Mainstream Enthusiasts": [
      {
        "timestamp_in_seconds": 1596805406,
        "weight": 1
      }
    ],
    "PC Purists": [
      {
        "timestamp_in_seconds": 1596805406,
        "weight": 1
      }
    ]
  },
```

**`Triggered ad`**

```
[3573:775:0807/090523.202556:VERBOSE1:ads_impl.cc(862)] Serving ad from categories:
[3573:775:0807/090523.202631:VERBOSE1:ads_impl.cc(864)]   gaming-gaming
[3573:775:0807/090523.202682:VERBOSE1:ads_impl.cc(864)]   personal finance-personal finance
[3573:775:0807/090523.202722:VERBOSE1:ads_impl.cc(864)]   technology & computing-technology & computing
[3573:775:0807/090523.202760:VERBOSE1:ads_impl.cc(864)]   PC Purists
[3573:775:0807/090523.202803:VERBOSE1:ads_impl.cc(864)]   Mainstream Enthusiasts
[3573:775:0807/090523.202841:VERBOSE1:ads_impl.cc(864)]   Alpha Influencers
[3573:775:0807/090523.205885:VERBOSE2:eligible_ads_priority_filter.cc(81)] 3 eligible ads with a priority of 1
[3573:775:0807/090523.205944:VERBOSE1:ads_impl.cc(970)] Found 3 eligible ads
[3573:775:0807/090523.205999:VERBOSE3:client.cc(545)] Saving client state
[3573:775:0807/090523.206081:VERBOSE3:client.cc(545)] Saving client state
[3573:775:0807/090523.206180:VERBOSE3:client.cc(545)] Saving client state
[3573:775:0807/090523.206243:VERBOSE3:client.cc(545)] Saving client state
[3573:775:0807/090523.206300:VERBOSE1:ads_impl.cc(1233)] Ad notification shown:
  uuid: e475c00e-df6b-4ebb-bde5-476cf6fc3e68
  parentUuid: 97bb553b-f18f-4e90-8ae3-66cd1cb52249
  creativeInstanceId: e5cc348a-d244-4540-a97e-a23899358d09
  creativeSetId: ed56ce80-afdf-412a-925a-54dfa5c316e1
  category: pc purists
  title: Intel Purchase Intent Test
  body: PC Purists
```

</details>

<details>
<summary> Case 2 - searching for `Gaming eSports Content` via https://www.google.com</summary>

```
[3706:775:0807/090654.814886:VERBOSE1:ads_impl.cc(732)] Extracting purchase intent signal from visited URL
```

**`Entry in client.json`**:

```
  "purchaseIntentSignalHistory": {
    "Alpha Influencers": [
      {
        "timestamp_in_seconds": 1596805614,
        "weight": 1
      }
    ]
  },
```

**`Triggered ad`**

```
[3706:775:0807/090812.812046:VERBOSE1:ads_impl.cc(862)] Serving ad from categories:
[3706:775:0807/090812.812117:VERBOSE1:ads_impl.cc(864)]   Alpha Influencers
[3706:775:0807/090812.813398:VERBOSE2:eligible_ads_priority_filter.cc(81)] 1 eligible ads with a priority of 1
[3706:775:0807/090812.813463:VERBOSE1:ads_impl.cc(970)] Found 1 eligible ads
[3706:775:0807/090812.813507:VERBOSE3:client.cc(545)] Saving client state
[3706:775:0807/090812.813575:VERBOSE3:client.cc(545)] Saving client state
[3706:775:0807/090812.813631:VERBOSE3:client.cc(545)] Saving client state
[3706:775:0807/090812.813686:VERBOSE3:client.cc(545)] Saving client state
[3706:775:0807/090812.813741:VERBOSE1:ads_impl.cc(1233)] Ad notification shown:
  uuid: 7ad2c723-ad08-4f23-9a2f-5df828225574
  parentUuid: 09ba9c4a-3836-466f-9e6f-7fe64e9c381b
  creativeInstanceId: ca630efe-df7e-4558-bbaf-002ad26f9625
  creativeSetId: 8ac894c3-928c-4096-98f9-617216d60d85
  category: alpha influencers
  title: Intel Purchase Intent Test
  body: Alpha Influencers
```

</details>

<details>
<summary> Case 3 - searching for "Customized PC" via https://www.google.com</summary>

```
[3746:775:0807/091010.095623:VERBOSE1:ads_impl.cc(732)] Extracting purchase intent signal from visited URL
```

**`Entry in client.json`**:

```
  "purchaseIntentSignalHistory": {
    "PC Purists": [
      {
        "timestamp_in_seconds": 1596805810,
        "weight": 1
      }
    ]
  },
```

**`Triggered ad`**

```
[3746:775:0807/091118.021389:VERBOSE1:ads_impl.cc(862)] Serving ad from categories:
[3746:775:0807/091118.021450:VERBOSE1:ads_impl.cc(864)]   PC Purists
[3746:775:0807/091118.022603:VERBOSE2:eligible_ads_priority_filter.cc(81)] 1 eligible ads with a priority of 1
[3746:775:0807/091118.022654:VERBOSE1:ads_impl.cc(970)] Found 1 eligible ads
[3746:775:0807/091118.022695:VERBOSE3:client.cc(545)] Saving client state
[3746:775:0807/091118.022756:VERBOSE3:client.cc(545)] Saving client state
[3746:775:0807/091118.022807:VERBOSE3:client.cc(545)] Saving client state
[3746:775:0807/091118.022857:VERBOSE3:client.cc(545)] Saving client state
[3746:775:0807/091118.022906:VERBOSE1:ads_impl.cc(1233)] Ad notification shown:
  uuid: 8b68d145-381e-4211-bbf6-fc45cc60178e
  parentUuid: de927498-f95b-46fa-b7d7-9dc408de1832
  creativeInstanceId: e5cc348a-d244-4540-a97e-a23899358d09
  creativeSetId: ed56ce80-afdf-412a-925a-54dfa5c316e1
  category: pc purists
  title: Intel Purchase Intent Test
  body: PC Purists
```

</details>

<details>
<summary> Case 4 - searching for "Gaming Advice" via https://www.google.com</summary>

```
[3778:775:0807/091259.485633:VERBOSE1:ads_impl.cc(732)] Extracting purchase intent signal from visited URL
```

**`Entry in client.json`**:

```
  "purchaseIntentSignalHistory": {
    "Mainstream Enthusiasts": [
      {
        "timestamp_in_seconds": 1596805979,
        "weight": 1
      }
    ]
  },
```

**`Triggered ad`**

```
[3778:775:0807/091414.044265:VERBOSE1:ads_impl.cc(862)] Serving ad from categories:
[3778:775:0807/091414.044333:VERBOSE1:ads_impl.cc(864)]   Mainstream Enthusiasts
[3778:775:0807/091414.045753:VERBOSE2:eligible_ads_priority_filter.cc(81)] 1 eligible ads with a priority of 1
[3778:775:0807/091414.045811:VERBOSE1:ads_impl.cc(970)] Found 1 eligible ads
[3778:775:0807/091414.045856:VERBOSE3:client.cc(545)] Saving client state
[3778:775:0807/091414.045928:VERBOSE3:client.cc(545)] Saving client state
[3778:775:0807/091414.045989:VERBOSE3:client.cc(545)] Saving client state
[3778:775:0807/091414.046046:VERBOSE3:client.cc(545)] Saving client state
[3778:775:0807/091414.046102:VERBOSE1:ads_impl.cc(1233)] Ad notification shown:
  uuid: 15a70b52-4b8f-454e-a9ae-268a573883d3
  parentUuid: 9fa2ac27-e572-49fe-a453-8c9edcaba5d2
  creativeInstanceId: f44b56d6-7a4e-4f8c-8187-b9861e1ac1da
  creativeSetId: fe78d034-6a6d-4bf5-acc7-7c4976289b70
  category: mainstream enthusiasts
  title: Intel Purchase Intent Test
  body: Mainstream Enthusiasts
```

</details>

-----------------
